queries/query/highlights.scm

```diff
@@ -1,12 +1,10 @@
 (string) @string
 (escape_sequence) @string.escape
-(capture) @type
-(anonymous_node) @string
+(capture (identifier) @type)
+(anonymous_node (identifier) @string)
```

Comment:
previously we were matching the whole parent node, it didn't look weird because the other queries had precedence